src/Storages/StorageURL.cpp

```diff
@@ -136,6 +148,7 @@ namespace
                                 timeouts,
                                 credentials,
                                 context->getSettingsRef().max_http_get_redirects,
+                                userInfo.length() == 0 ? Poco::Net::HTTPBasicCredentials{} : Poco::Net::HTTPBasicCredentials(ostr.str()),
```

Comment:
A little strange, this is now not consistent with ReadWriteBufferFromHTTP constructor, I think build check will fail now.
Probably merged with master incorrectly?

Reply:
OK, I will chang to if...else...patten.

Reply:
No this is what I meant, credentials arguments is passed 2 lines above. See https://github.com/ClickHouse/ClickHouse/blob/341f705419811a5690e9e87ebfbddfb8282925c3/src/IO/ReadWriteBufferFromHTTP.h#L466-L473.

Reply:
Got it! It was a mistake of merge codes. I will push again.